RobotCrawler/src/main/java/parser/kobobooks/KoboPageDownloader.java

```diff
@@ -0,0 +1,36 @@
+package parser.kobobooks;
+
+import org.jsoup.Connection;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+
+import java.io.IOException;
+import java.util.Objects;
+import java.util.Optional;
+
+public class KoboPageDownloader {
+
+    private static final String baseUrl = "https://store.kobobooks.com";
+    private static Connection.Response login;
+
+    public Optional<Document> getDocument(String additionalUrl) {
+        Document document = null;
+        try {
+            if(Objects.isNull(login)) {login();}
+            document = Jsoup.connect(baseUrl.concat(additionalUrl))
```

Comment:
I've removed this class and now I'm using DocumentBuilder.
Also I've added some additional behavior to DocumentBuilder